recipes/pycutest/activate.sh

```diff
@@ -0,0 +1,18 @@
+share_path=${CONDA_PREFIX}/share
+
+export ARCHDEFS=${share_path}/archdefs
+export SIFDECODE=${share_path}/sifdecode
+export CUTEST=${share_path}/cutest
+export MASTSIF=${share_path}/mastsif
+
+export MYARCH="pc64.lnx.gfo"
+
+# backup initial standard variables to restore them in deactivate.sh
+export _CONDA_PYCUTEST_OLD_PATH=${PATH}
+export _CONDA_PYCUTEST_OLD_MANPATH=${MANPATH}
+
+export PATH=${SIFDECODE}/bin:${PATH}
+export PATH=${CUTEST}/bin:${PATH}
```

Comment:
Why aren't the binaries installed in  `${PREFIX}/bin`? I would rather not touch `$PATH`.

Reply:
I tried to get pycutest to work by installing things in the standard locations, but unfortunately there are hardcoded paths that rely on the executable to be installed in `${SIFDECODE}/bin` and `${CUTEST}/bin`.